lib/Inst/Inst.cpp

```diff
@@ -285,6 +285,17 @@ std::string Inst::getMoreKnownBitsString(bool NonZero, bool NonNegative, bool Po
   return Str;
 }
 
+std::string Inst::getDemandedBitsString(llvm::APInt DBVal) {
+  std::string Str;
+  for (int K=DBVal.getBitWidth()-1; K>=0; --K) {
```

Comment:
can you put spaces around the binary operators such as "-" and ">="? That is the LLVM style and I think we are doing it consistently elsewhere. don't do this for unary operators.